static/templates/actors/hazard/partials/sidebar.hbs

```diff
@@ -53,12 +53,13 @@
             <div class="sidebar-saves">
                 {{#each saves as |save|}}
                     <div class="save">
-                        <h2 {{#if save.mod includeZero=true}}class="save-name{{#if @root.options.editable}} rollable{{/if}}" data-save="{{save.type}}"{{/if}}>
-                            {{#if save.mod includeZero=true}}
-                                <i class="fa-solid fa-dice-d20"></i>
-                            {{/if}}
-                            {{localize save.label}}
-                        </h2>
+                        {{#if save.mod includeZero=true~}}
+                            <a
+                                class="rollable"
+                                data-action="roll-check"
+                                data-statistic="{{save.type}}"
+                            ><i class="fa-solid fa-dice-d20"></i> {{localize save.label}}</a>
+                        {{~/if}}
```

Comment:
Can you still add new saves in edit mode after this? I believe those become null instead, but I don't remember.

Reply:
Tested and still works